eng/scripts/Test-Packages.ps1

```diff
@@ -30,22 +31,24 @@ if ($PackageInfoPath) {
   | ConvertFrom-Json
 }
 else {
-  $packagesToTest = Get-AllPackagesInRepo
+  . "$PSScriptRoot\Language-Settings.ps1"
```

Comment:
Line 14's invocation of eng/common/scripts/common.ps1 will import Language-Settings.ps1